include/mfmg/dealii/dealii_matrix_free_mesh_evaluator.hpp

```diff
@@ -56,6 +56,16 @@ class DealIIMatrixFreeMeshEvaluator : public DealIIMeshEvaluator<dim>
    */
   virtual ~DealIIMatrixFreeMeshEvaluator() override = default;
 
+  /**
+   * Create a deep copy of this class such that initializing on another
+   * agglomerate works.
```

Comment:
Please add a note that this was introduced because calls to member functions `matrix_free_initialize_agglomerate()` and `matrix_free_evaluate_agglomerate()` (implemented in user provided class deriving from `MatrixFreeMeshEvaluator`) are not thread safe and that it is not the only option to solve the problem.